examples/00-fluent/modeling_cavitation.py

```diff
@@ -0,0 +1,359 @@
+"""
+
+Modeling Cavitation
+-------------------
+This example examines the pressure-driven cavitating flow of water through a
+sharp-edged orifice. This is a typical configuration in fuel injectors, and
+brings a challenge to the physics and numerics of cavitation models because of
+the high pressure differentials involved and the high ratio of liquid to vapor
+density.
+
+This example uses the multiphase modeling capability of Ansys Fluent, and will
+be able to predict the strong cavitation near the orifice after flow separation
+at a sharp edge.
+
+**Workflow tasks**
+
+This example demonstrates how to do the following:
+
+- Set boundary conditions for internal flow
+- Use the mixture model with cavitation effects
+- Calculate a solution using the pressure-based coupled solver
+
+**Problem description**
+
+The problem considers the cavitation caused by the flow separation after a
+sharp-edged orifice. The flow is pressure driven, with an inlet pressure of 5 x
+10^5 Pa and an outlet pressure of 9.5 x 10^4 Pa. The orifice diameter is 4 x
+10^-3 m, and the geometrical parameters of the orifice are D/d = 2.88 and L/d =
+4, where D, d, and L are the inlet diameter, orifice diameter, and orifice
+length respectively.
+"""
+
+###############################################################################
+# Example Setup
+# -------------
+# Before you can begin, you must set up the example and initialize this
+# workflow.
+#
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports, which includes downloading and importing
+# the geometry file.
+
+# sphinx_gallery_thumbnail_path = '_static/cavitation_model.png'
+
+import ansys.fluent.core as pyfluent
```

Comment:
```suggestion
# sphinx_gallery_thumbnail_path = '_static/cavitation_model.png'
import ansys.fluent.core as pyfluent
```
These two lines need to be in the same "block" with no lines in between, otherwise Sphinx will consider them two different blocks and display the `# sphinx_gallery_thumbnail_path = '_static/cavitation_model.png'` config command in the webpage output as described [here](https://sphinx-gallery.github.io/stable/configuration.html#providing-thumbnail).

Reply:
Ok I will keep them together.